app/assets/stylesheets/content/_badges.md

```diff
@@ -0,0 +1,13 @@
+# Badges
```

Comment:
How about "indicator" or "count"?

On Wednesday, April 22, 2015, Jan Sandbrink <notifications@github.com>
wrote:

> In app/assets/stylesheets/content/_badges.md
> <https://github.com/opf/openproject/pull/2891#discussion_r28848235>:
>
> > @@ -0,0 +1,13 @@
> > +# Badges
>
> I don't know where we will use it.
>
> Where it is used now, I would definitely suggest a different name, but
> that might also be because I have associations with the word "badge"...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/opf/openproject/pull/2891/files#r28848235>.
>